webpack.config.cli.prod.js

```diff
@@ -55,4 +55,14 @@ prodConfig.module.rules.push({
   }),
 });
 
+// Remove all data-test or data-test-* attributes
+const babelLoaderConfigIndex = prodConfig.module.rules.findIndex((rule) => {
+  return rule.loader === 'babel-loader';
+});
+prodConfig.module.rules[babelLoaderConfigIndex].options.plugins = [
```

Comment:
This seems like it would override previously defined babel plugins. 

Maybe something like this?

```js
const babelLoaderConfig = prodConfig.module.rules.find(rule => rule.loader === 'babel-loader');

babelLoaderConfig.options.plugins = (babelLoaderConfig.options.plugins || []).concat([
  [require.resolve('babel-plugin-remove-jsx-attributes'), {
    patterns: ['^data-test.*$']
  }]
]);
```

Reply:
Oops, you're right, that would overwrite. Changed 👌